### ISSUE TITLE
[BUG] Group Separator not working correctly with kirby-input setMaxOnOverflow in danish locale

### ISSUE THREAD

**Issue by ckibsen** [kirbydesign/designsystem#3611]
- [ x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
Using kirby-input with kirby-decimal-mask and setMaxOnOverflow = true, fails to correcly format numbers when running in danish locale. 

## Describe how to reproduce the bug

Setting value = 120000.25 and max = 160001.25 in kirby-input field. 

When using english locale, typing a new value of eg. 161000 results in the value being replaced by the max value 160,001.25 (nicely formatted).

When using danish locale, typing a new value of 161000 doesn't trigger a change. The same is the case if ,00 is typed as decimals - no effect. Typing a way to large number like eg 161000000 results in the input being replaced with 16.000.125 (formattet, but way larger than the max allowed value). 

## Which Kirby version was used?
9.6.0 

## What was the expected behavior?
When running in danish locale, I expect 161000 to be replaced by the formatted max value 160.000,25.

## Add any screenshots
Added two screen recordings in danish and english, to show the different behaviours.

## Please complete the following information:
- OS: macOS
- Browser: Chrome
- Browser version: 128.0.6613.138

## Are there any additional context?
Rasmus Kjeldgaard has acknowledged there is a bug when using the group-separator in conjunction with kirby-input / kirby-decimal-mask, and danish locale. 

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

https://github.com/user-attachments/assets/8a08c1e4-0231-427f-8885-4366ec7c3f1a

https://github.com/user-attachments/assets/f3f9d5da-10e9-4c42-8c8b-b1689ec1acc3

**Comment by ckibsen** [comment]
https://github.com/user-attachments/assets/e78d68f8-6a58-47fa-bc3b-c970efae26a9


https://github.com/user-attachments/assets/3e5cfad9-d67f-4c08-a299-b624a9ccb44f

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
